src/gui/MenuWidgets.cpp

```diff
@@ -893,9 +893,7 @@ void MenuPage::Render() {
 			default:
 			{
 				if(GInput->getMouseButtonNowPressed(Mouse::Button_0)) {
-					Widget * widget = m_children.GetZoneWithID(BUTTON_MENUOPTIONS_CONTROLS_CUST_DEFAULT);
-					
-					if(widget == m_selected) {
+					if(m_selected->GetZoneWithID(BUTTON_MENUOPTIONS_CONTROLS_CUST_DEFAULT)) {
```

Comment:
Your new code will match any widgets *containing* the "Default" button. Not sure if these can ever be "selected" though but it would be safer to just check the id.